lib/filterx/filterx-eval.h

```diff
@@ -45,6 +52,7 @@ struct _FilterXEvalContext
   FilterXError error;
   LogTemplateEvalOptions template_eval_options;
   GPtrArray *weak_refs;
+  FilterXEvalControl eval_control_type;
```

Comment:
[nitpick]

`eval_control_override` or `eval_control_modifier` feels better to me.